tests/Xaki.Tests/ObjectLocalizerFacts.cs

```diff
@@ -19,15 +19,26 @@ public abstract class TestBase
             };
         }
 
+        public class GetEmptyJsonString : TestBase
+        {
+            [Fact]
+            public void ReturnsCorrectJsonStringForSupportedLanguages()
+            {
+                var actual = ObjectLocalizer.GetEmptyJsonString();
+
+                Assert.Equal("{\"en\":\"\",\"xx\":\"\"}", actual);
+            }
+        }
+
         public class Serialize : TestBase
         {
             [Fact]
             public void ReturnsCorrectJsonForValidContent()
             {
                 var contents = new Dictionary<string, string>
                 {
-                    {Constants.LanguageCode1, Constants.AnyString1 },
-                    {Constants.LanguageCode2, Constants.AnyString2 }
+                    [Constants.LanguageCode1] = Constants.AnyString1,
+                    [Constants.LanguageCode2] = Constants.AnyString2
```

Comment:
I do like the C#6 dictionary initializer pattern better, need to get it into muscle memory